### ISSUE TITLE
Error deleting multiple records 

### ISSUE THREAD

**Issue by rocurley** [go-pg/pg#810]
Hey,

When updating go-pg, we found that deleting multiple records yields:
`ERROR #42P01 missing FROM-clause entry for table "_data"`
This appears to have been introduced in 8a8f5d78f8b3610cf48149ac6022c0942c7aed40 .

You can reproduce this using the following code (which I used along with `git bisect` to find the above commit).

```
package pg_test

import (
	"testing"

	"github.com/go-pg/pg"
	"github.com/go-pg/pg/orm"
)

var delTestDb *pg.DB

func init() {
	delTestDb = connect()
}

func Test_CreateTable(t *testing.T) {
	type Foo struct {
		Id int
	}

	err := delTestDb.CreateTable(&Foo{}, &orm.CreateTableOptions{Temp: true})
	if err != nil {
		t.Error(err)
	}

	for i := 0; i < 10; i++ {
		err = delTestDb.Insert(&Foo{})
		if err != nil {
			t.Error(err)
		}
	}

	var foos []Foo
	err = delTestDb.Model(&foos).Limit(5).Select()
	if err != nil {
		t.Error(err)
	}

	if l := len(foos); l != 5 {
		t.Errorf("Expected 5 results for 1st select but got %v", l)
	}

	err = delTestDb.Delete(&foos)
	switch err {
	case nil:
	case pg.ErrMultiRows:
	default:
		t.Error(err)
	}
	var foosRest []Foo
	err = delTestDb.Model(&foosRest).Select()
	if err != nil {
		t.Error(err)
	}
	if l := len(foosRest); l != 5 {
		t.Errorf("Expected 5 results for 2nd select but got %v", l)
	}
}
```

**Comment by vmihailenco** [comment]
My understanding is that bulk deletes were never intentionally supported. There are no commits, tests, examples and/or entries in readme. I guess it could work using code for bulk-updates, but it was a lucky coincidence.

I am going to add support for bulk-deletes, but you will have to test your code that everything works as expected.

**Comment by vmihailenco** [comment]
Should be fixed by https://github.com/go-pg/pg/pull/812 and released in v6.11.0
